### PR TITLE
feat(theme): btn variations support into navbar

### PR DESCRIPTION
**Pull request by amaalej**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
Talend projects manage the height, min-height and margin for the buttons inside navbar 

**What is the new behavior?**
As the buttons style are the same for the Talend's projects, they're managed by theme.

**Does this PR introduce a breaking change?**
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

File: packages/theme/src/theme/_navbar.scss [hunk 1]
```diff
@@ -2,6 +2,27 @@
...
+
+		.btn {
+			&-default,
+			&-primary,
+			&-success,
+			&-info,
+			&-warning,
+			&-danger {
+				min-height: inherit;
+				height: auto;
```
Comment (frassinier) [inline packages/theme/src/theme/_navbar.scss]: :hover and :focus states need to be reset

File: packages/theme/src/theme/_navbar.scss [hunk 2]
```diff
@@ -2,6 +2,27 @@
 .navbar {
 	border: none;
 
+	.navbar-nav {
```
Comment (frassinier) [inline packages/theme/src/theme/_navbar.scss]: `&-nav {` instead

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

File: output/theme.sasslint.txt [hunk 3]
```diff
@@ -23,10 +23,16 @@ src/theme/_forms.scss
...
+  13:1  warning  Space expected between blocks  empty-line-between-blocks
+
 src/theme/_tooltip.scss
   2:3  warning  Mixed tabs and spaces  indentation
 
-✖ 13 problems (11 errors, 2 warnings)
+src/theme/_variables.scss
+  158:34  warning  Hex notation should all be upper case  hex-notation
+
+✖ 15 problems (11 errors, 4 warnings)
```
Comment (charlesnguyen) [inline output/theme.sasslint.txt]: fix lint errors pls

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://349.talend.surge.sh)

**Pull request merged**